taxi/forms.py

```diff
@@ -0,0 +1,42 @@
+from django.contrib.auth.forms import UserCreationForm
+from django.contrib.auth import get_user_model
+from django import forms
+from .models import Driver, Car

```

Comment:
Use absolute imports instead of relative ones here and in the other places as well